teko.go

```diff
@@ -2,14 +2,17 @@ package main
 
 import (
   "github.com/cstuartroe/teko/src/lexparse"
+  "fmt"
+  "os"
 )
 
 func main() {
-  l := lexparse.Line{
-    Num: 45,
-    Value: "let x = 3;",
-    Filename: "/this/is/not/a/file.to",
+  if len(os.Args) != 2 {
+    fmt.Println("Please supply exactly one argument, the filename")
+    os.Exit(0)
```

Comment:
You probably want to exit with a non-zero code here to indicate an error.

Reply:
True, although eventually this will be a REPL so it's probably not a priority

Are non-zero status codes used to indicate user error? For some reason I assumed they were just for unexpected runtime errors.

Reply:
Yeah, any time the program did not exit successfully you should return a non-zero exit code. I tend to think of it as, if I were writing a bash script that called this program, would I expect the script to crash or continue in this case? In the case of incorrect command-line arguments, I would definitely expect it to crash.